core/src/main/c/freebsd/kqueue.c

```diff
@@ -27,88 +27,168 @@
 }
 
-JNIEXPORT jshort JNICALL Java_io_questdb_network_KqueueAccessor_getFilterOffset
+JNIEXPORT jshort JNICALL Java_io_questdb_KqueueAccessor_getFilterOffset
         (JNIEnv *e, jclass cl) {
     return (short) offsetof(struct kevent, filter);
 }
 
-JNIEXPORT jshort JNICALL Java_io_questdb_network_KqueueAccessor_getDataOffset
+JNIEXPORT jshort JNICALL Java_io_questdb_KqueueAccessor_getDataOffset
         (JNIEnv *e, jclass cl) {
     return (short) offsetof(struct kevent, udata);
 }
 
-JNIEXPORT jshort JNICALL Java_io_questdb_network_KqueueAccessor_getFlagsOffset
+JNIEXPORT jshort JNICALL Java_io_questdb_KqueueAccessor_getFlagsOffset
         (JNIEnv *e, jclass cl) {
     return (short) offsetof(struct kevent, flags);
 }
 
-JNIEXPORT jshort JNICALL Java_io_questdb_network_KqueueAccessor_getEvAdd
+JNIEXPORT jshort JNICALL Java_io_questdb_KqueueAccessor_getEvAdd
         (JNIEnv *e, jclass cl) {
     return EV_ADD;
 }
 
-JNIEXPORT jshort JNICALL Java_io_questdb_network_KqueueAccessor_getEvOneshot
+JNIEXPORT jshort JNICALL Java_io_questdb_KqueueAccessor_getEvOneshot
         (JNIEnv *e, jclass cl) {
     return EV_ONESHOT;
 }
 
-JNIEXPORT jshort JNICALL Java_io_questdb_network_KqueueAccessor_getEvDelete
+JNIEXPORT jshort JNICALL Java_io_questdb_KqueueAccessor_getEvDelete
         (JNIEnv *e, jclass cl) {
     return EV_DELETE;
 }
 
-JNIEXPORT jint JNICALL Java_io_questdb_network_KqueueAccessor_kqueue
+JNIEXPORT jshort JNICALL Java_io_questdb_KqueueAccessor_getEvClear
+        (JNIEnv *e, jclass cl) {
+    return EV_CLEAR;
+}
+
+JNIEXPORT jint JNICALL Java_io_questdb_KqueueAccessor_kqueue
         (JNIEnv *e, jclass cl) {
     return kqueue();
 }
 
-JNIEXPORT jint JNICALL Java_io_questdb_network_KqueueAccessor_kevent
+JNIEXPORT jint JNICALL Java_io_questdb_KqueueAccessor_kevent
         (JNIEnv *e, jclass cl, jint kq, jlong changelist, jint nChanges, jlong eventlist, jint nEvents, jint timeout) {
     int tv_sec = timeout / 1000;
     struct timespec _timeout = {tv_sec, (timeout - tv_sec * 1000) * 1000 * 1000};
     return (jint) kevent(
-            kq,
-            (const struct kevent *) changelist,
+            kq, (const struct kevent *) changelist,
             nChanges,
             (struct kevent *) eventlist,
             nEvents,
             &_timeout
     );
 }
 
-JNIEXPORT jlong JNICALL Java_io_questdb_network_KqueueAccessor_pipe
+JNIEXPORT jint JNICALL Java_io_questdb_KqueueAccessor_keventRegister
+        (JNIEnv *e, jclass cl, jint kq, jlong changelist, jint nChanges) {
+    return (jint) kevent(
+            kq,
+            (const struct kevent *) changelist,
+            nChanges,
+            NULL,
+            0,
+            NULL
+    );
+}
+
+JNIEXPORT jint JNICALL Java_io_questdb_KqueueAccessor_keventGetBlocking
+        (JNIEnv *e, jclass cl, jint kq, jlong eventList, jint nEvents) {
+    int res;
+    RESTARTABLE(kevent(kq, NULL, 0, (struct kevent *) eventList, nEvents, NULL), res);
```

Comment:
what is happening with "res" here? it is passed by value and then returned, it is always 0